integration_tests/integration/prisonerReportReview.spec.ts

```diff
@@ -321,4 +359,19 @@ context('Prisoner report - reviewer view', () => {
     PrisonerReportPage.reviewSubmit().click()
     cy.get('*[class^="govuk-error-message"]').contains('A reason is required')
   })
+  it('should not contain the review panel if status is RETURNED', () => {
```

Comment:
Let's just hold this off for a minute... we need to check this with the designers as well

Reply:
Approved!